### ISSUE TITLE
Installing go tools must print the import path

### ISSUE THREAD

**Issue by ahmetb** [Microsoft/vscode-go#1029]
```
Installing 1 tool
  impl

Installing impl SUCCEEDED

All tools successfully installed. You're ready to Go :).
```

This doesn't help me. I maintain a separate `install_gotools.sh` script that I run on all my workstations to install go development tools (incl. the ones vscode-go uses). I think this operation should print the full package import path while it's doing `go get`, not just the package name.

**Comment by ramya-rao-a** [comment]
Fair point

**Comment by ramya-rao-a** [comment]
The fix is now out in the latest update (0.6.63) to the Go extension
